pkg/httpd/websftp.go

```diff
@@ -101,7 +101,7 @@ type webSftpRequest struct {
 	IsDir   bool   `json:"is_dir"`
 }
 
-func (h *webSftp) dispatch(msg *Message) {
+func (h *webSftp) dispatch(msg Message) {
 	message := Message{
 		Id:          msg.Id,
 		Cmd:         msg.Cmd,
```

Comment:
There haven't been any changes made to this code between version 2021-09-01 and 2025-04-15. It appears there is no need for any adjustments at present.